doc/api/http.md

```diff
@@ -3720,9 +3720,11 @@ events will be emitted in the following order:
 Setting the `timeout` option or using the `setTimeout()` function will
 not abort the request or do anything besides add a `'timeout'` event.
 
-Passing an `AbortSignal` and then calling `abort` on the corresponding
+Passing an `AbortSignal` and then calling `abort()` on the corresponding
 `AbortController` will behave the same way as calling `.destroy()` on the
-request itself.
+request. Specifically, the `'error'` event will be emitted with an error with
```

Comment:
Nit
```suggestion
request itself. Specifically, the `'error'` event will be emitted with an error with
```

Reply:
Was going for brevity. "Itself" doesn't seem to add clarity. What do you think?

Reply:
I dont think its a necessary change to this pr, Id leave it as it was previously